### PR TITLE
James 1857: JWS-Create James Image for get started part

### PR DESCRIPTION
**Pull request by quynhn**
It depends on JAMES-1842: JPA and Lucene should be default configuration

### CODE REVIEW

**Comment by quynhn** [comment]
I changed as your guide and committed

**Comment by aduprat** [comment]
Ok, sorry i've misunderstood...
The based image is from `dockerfiles/run/spring/Dockerfile`
I recommend you to rename the folder `custom` to `provisioned` as it's the only added value.
Then you may add some comment in the README.

**Comment by quynhn** [comment]
I updated as your comment. Thanks Antoine

File: dockerfiles/run/spring/provisioned/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Run James
+#
+# VERSION	1.0
+
+FROM quynhnn/james-project-jpa
```
Comment (mbaechler) [inline dockerfiles/run/spring/provisioned/Dockerfile]: should not use quynhnn in the image name

Comment (chibenwa) [inline dockerfiles/run/spring/provisioned/Dockerfile]: I guess it is for testing purposes

File: README.adoc [hunk 2]
```diff
@@ -196,6 +196,9 @@ $ keytool -genkey -alias james -keyalg RSA -keystore dockerfiles/run/spring/dest
 Then we need to build james container :
 $ docker build -t james_run dockerfiles/run/spring/
 
+If we need to build james container with the default initial data (initial domain: james.local and initial users: user01, user02, user03):
```
Comment (chibenwa) [inline README.adoc]: You should indicate it is built from a pre-built jpa james server...

**Comment by chibenwa** [comment]
If we do so, we need to update jenkins in order to build and push java6 docker image to docker hub. This PR needs to be marged after the one on the CI.

File: dockerfiles/run/spring/provisioned/Dockerfile (hunk 1 above)
Comment (quynhn) [inline dockerfiles/run/spring/provisioned/Dockerfile]: Yes, it is and when this PR is validated and get approval, we have to push the base image to Linagora docker reporsitoris

**Comment by quynhn** [comment]
`If we do so, we need to update jenkins in order to build and push java6 docker image to docker hub. This PR needs to be marged after the one on the CI.`
Yes, guide me how to do that.

**Pull request merged**